src/Common/src/TypeSystem/IL/Stubs/PInvokeILEmitter.cs

```diff
@@ -460,6 +460,32 @@ private TypeDesc MarshalReturnValue(TypeDesc type)
                 return type;
             }
 
+            //
+            // Work-around to enable a p/invoke with struct return value. When compiling the stub for
+            // System.Console's Interop.mincore.GetLargestConsoleWindowSize, the JIT decides it cannot
+            // generate an inline p/invoke because the return type is a struct.
+            //
+            if (type.IsValueType && !type.IsPrimitive)
```

Comment:
Can you make sure there's an issue for this (and the issue # is linked from here) per Jan's comment?